bootbind

```diff
@@ -6,13 +6,13 @@ set -e
 shopt -s dotglob
 shopt -s lastpipe
 
-if mountpoint -q /boot; then
-	echo "/boot is already mounted, aborting..." 1>&2
+if mountpoint -q ${ESP_ROOT}; then
```

Comment:
Is $ESP_ROOT supposed to be defined by the OS? It doesn't appear to be defined in the scope of systemd units on my Fedora Linux system:

```
# systemd-run --pipe /usr/bin/env
Running as unit: run-u99.service
LANG=en_US.UTF-8
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin
INVOCATION_ID=f1485bedd3654398926cfb17deb94a1d
SYSTEMD_EXEC_PID=5702
TERM=xterm-256color
```

Reply:
I see now. ${ESP_ROOT} is being substituted by the `envsubs` command in the makefile. That's a very neat trick. 🙂